guides/hack/08-generic-types-and-functions/20-reified-generics.md

```diff
@@ -0,0 +1,187 @@
+# Reified Generics
+
+## Introduction
```

Comment:
Are they enabled by default? If so, since when - if not:
- how are they enabled?
- add a note strongly discouraging use of opt-in features (as it means that any dependents must use the same opt-in)